lib/app/middleware.js

```diff
@@ -3,10 +3,8 @@ let files = require.context('@/<%= dir.middleware %>', false, /^\.\/(?!<%= ignor
 let filenames = files.keys()
```

Comment:
All `let`s in this file (5 of them) can be replaced by `const`

Reply:
Also, for line 13:
```js
const PREFIX_EXT_EX = /^\.\/|\.(<%= extensions %>)$/g
for (const filename of filenames) {
  const name = filename.replace(PREFIX_EXT_EX, "")
  middleware[name] = getModule(filename)
}
```
Allegedly a global flag is better than two invocations,  but in any case this regex should be outside the loop.

Reply:
Will do